test/e2e/tests/tokens/nft/erc1155-interaction.spec.js

```diff
@@ -65,6 +75,45 @@ describe('ERC1155 NFTs testdapp interaction', function () {
           true,
           `transaction item should be displayed in activity tab`,
         );
+        await driver.clickElement('[data-testid="activity-list-item-action"]');
+        await driver.clickElement({
+          text: 'View on block explorer',
+          tag: 'a',
+        });
+        // Switch to block explorer
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle('E2E Test Page', windowHandles);
+        await driver.findElement('[data-testid="empty-page-body"]');
+        // Verify block explorer
+        await driver.waitForUrl({
+          url: 'https://etherscan.io/tx/0xfe4428397f7913875783c5c0dad182937b596148295bc33c7f08d74fdee8897f',
+        });
+        // switch to Dapp
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          WINDOW_TITLES.TestDApp,
+          windowHandles,
+        );
+        await driver.fill('#watchAssetInput', '1');
+        await driver.clickElement('#watchAssetButton');
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          WINDOW_TITLES.Dialog,
+          windowHandles,
+        );
+        await driver.clickElement('[data-testid="page-container-footer-next"]');
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          WINDOW_TITLES.ExtensionInFullScreenView,
+          windowHandles,
+        );
+        await driver.findClickableElement(
+          '[data-testid="account-overview__nfts-tab"]',
+        );
+        await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
```

Comment:
The test is flaking here.

Reply:
the problem is that you are trying to click on the NFT button but there is the previous popup for going to Etherscan open (see pic below).
You could either wait for the popup element to disappear (assertElementnotPresent) assuming this is closed automatically after adding the NFT, which I think it is, at least I see this on my local. So you can do:

```
await driver.assertElementNotPresent('[data-testid="popover-close"]');
await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
```

Alternatively we could close it if it's present using clickElementSafe:
```
await driver.clickElementSafe('[data-testid="popover-close"]');
await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
``

![image](https://github.com/user-attachments/assets/ad55a265-8424-44dd-9631-1c10b5a38859)

Reply:
Thank you @seaona ❤️